src/System.Management.Automation/resources/FormatAndOut_format_xxx.resx

```diff
@@ -179,4 +179,7 @@
   <data name="FormattingError" xml:space="preserve">
     <value>Failed to interpret format string "{0}".</value>
   </data>
+  <data name="SmartToStringError" xml:space="preserve">
+    <value>Object skipped: {0}.</value>
+  </data>
```

Comment:
This has been created, but is it used anywhere? If so, is it possible to provide a more context-informative error message? If not, this should be removed.

Reply:
Left over that needs to be removed